### DISCUSSION TITLE
Error - user.name not found

### DISCUSSION THREAD

**Discussion by ChrisPerlego** [extrawurst/gitui#975] [Q&A] [answered]
Hey all, I've just set up Gitui on a new Macbook and am using SSH to clone, push and manage my repos.

Since moving to the new machine, I've started seeing this error:

![image](https://user-images.githubusercontent.com/62247129/140055595-97d8b5fe-07a0-4b95-b308-6e287f8544ee.png)

I can commit without any issues using `git commit -m` in terminal, so I'm confused why I'd be seeing this in Gitui all of a sudden.

Does anyone know why my local git user name isn't found? How can I configure this?

**Comment by ChrisPerlego** [comment, marked as answer]
Just fixed it myself by following solution in https://stackoverflow.com/a/30182131 

Basically had to set user.name and user.email myself for some reason

**Reply by extrawurst** [reply]
Yeah this needs to be configured, maybe on your old machine this was done long time ago once or implicit by some git GUI application.

**Reply by ChrisPerlego** [reply]
I had the old machine for a year or so, so hard to know when it might've happened.

By the way, I'm getting an error when pushing from GITUI. It wasn't an issue on my old machine, and isn't an issue when calling `git push` in terminal

![image](https://user-images.githubusercontent.com/62247129/140763883-58bd9561-4520-47cf-9833-47aaac082855.png)

**Reply by extrawurst** [reply]
maybe https://github.com/extrawurst/gitui/issues/495

**Reply by ChrisPerlego** [reply]
Yep, fixed using https://github.com/extrawurst/gitui/issues/495#issuecomment-854948221
